src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html

```diff
@@ -0,0 +1,28 @@
+<div data-ng-switch="transclusionId">
+    <div data-ng-switch-when="menu">
+        <div class="moving-column-tab">
+            <i class="icon-document moving-column-icon"></i>
```

Comment:
Now that I look closer into this, it seems like the designs do not even have icons here. Strange.